pkg/reconciler/openshift/tektonresult/extension_test.go

```diff
@@ -21,7 +21,9 @@ import (
 	"path"
 	"testing"
 
+	"github.com/aws/smithy-go/ptr"
```

Comment:
earlier in the PR, there was import of the knative.dev/pkg/ptr package and it looks to me they provide the same functionality. Are they? Can we stick with just one of them, presumably knative.dev/pkg/ptr?